### DISCUSSION TITLE
[Projects Beta] Sync between projects' records

### DISCUSSION THREAD

**Discussion by yinonov** [community/community#8130] [Projects and Issues]
Enable in-between-projects binding or reference so that records in 1 project can be synced with records (multiple) in another project.
This will result in enabling an organization tree flow of top level `objectives > key results > epics > issues` maintained in different scopes, thus different projects.

reference:
[Airtable Sync](https://support.airtable.com/hc/en-us/articles/360052521254)

**Comment by luwangshell** [comment]
We need this feature as well! A roll-up view from multiple projects can really help us to get a nice overall view without duplicated effort.

**Comment by dralley** [comment]
This *might* be covered by https://github.com/github/roadmap/issues/274 ?

Although your suggestion sounds more like "let Project A inherit all of the records of Project B" than "let Project A possess Project B as if it were an item of its own".  But there's not really enough detail to know exactly what they mean.

**Reply by yinonov** [reply]
In Airtable it's possible to lookup records in other tables and compute some calculation. It's very efficient.
As described by the issue you mentioned, it would help _build up an explorable graph of work_
